### PR TITLE
Stefan jum/add dev content

### PR DESCRIPTION
**Pull request by StefanJum**

### CODE REVIEW

**Comment by razvand** [comment]
Please also add an update to the `README.md` file. See the updates from [this commit](https://github.com/unikraft/app-elfloader/commit/450a52101b17ed041ae97b9d219c997669309cdf).

**Comment by StefanJum** [comment]
Done @razvand

File: README.md [hunk 1]
```diff
@@ -188,22 +188,67 @@ Follow the steps below for the setup:
      10 directories, 7 files
      ```
 
+## Scripted Building and Running
+
+To make it easier build and run and test different configurations, the repository provides a set of scripts that do everything required.
```
Comment (razvand) [inline README.md]:
```suggestion
To make it easier to build, run and test different configurations, the repository provides a set of scripts that do everything required.
```

File: README.md [hunk 2]
```diff
@@ -188,22 +188,67 @@ Follow the steps below for the setup:
...
+These are scripts used for building different configurations of the helloworld application and for running these with all the requirements behind the scenes.
+
+First, generate these scripts by running the `generate.py` script while on the `helloworld` directory:
```
Comment (razvand) [inline README.md]:
```suggestion
First of all, grab the [`generate.py` script](https://github.com/unikraft/app-testing/blob/staging/scripts/generate.py) and place it in the `scripts/` directory by running:

'''console
wget https://raw.githubusercontent.com/unikraft/app-testing/staging/scripts/generate.py -O scripts/generate.py
chmod a+x scripts/generate.py
'''

Now, run the `generate.py` script.
You must run it in the root directory of this repository:
```

Replace the apostrophes above with backticks.

**Review by razvand — CHANGES_REQUESTED** [review]
Remove the `generate.py` script, since it's now downloaded. Also ignore it.

**Comment by razvand** [comment]
Also update the TLDR with the downloading and using of `generate.py`, same as documented [here](https://github.com/unikraft/app-llama2-c/#quick-set-up-aka-tldr).

**Review by razvand — APPROVED** [review]
Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request merged**